### PR TITLE
Add Terminals Are Sexy

### PR DESCRIPTION
**Pull request by k4m4**
## Terminals Are Sexy

**[https://github.com/k4m4/terminals-are-sexy](https://github.com/k4m4/terminals-are-sexy)**

> **A curated list of Terminal frameworks, plugins & resources for CLI lovers.**

GitHub Repo: [here](https://github.com/k4m4/terminals-are-sexy/)
**Project Site: [here](http://terminalsare.sexy/)**

(**NOTE**: I have included a [travis-ci](https://travis-ci.org/k4m4/terminals-are-sexy) badge, implemented to run with the [awesome_bot](https://github.com/dkhamsing/awesome_bot) for link validation purposes.)

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by legodude17** [comment]
Not followed:
 * Drop all the `A` / `An` prefixes in the descriptions.
 * Has an appropriate license (Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc) (If you use a license badge, it should be SVG, not PNG)
 * Does not include a travis badge.

Also, make sure to linkify all of the parts of the table of contents and add the topics `awesome`, `awesome-list`, and `list`.

**Comment by k4m4** [comment]
All done.
- [X] Drop all the A / An prefixes in the descriptions.
- [X] Has an appropriate license (Meaning something like CC0, NOT a code licence like MIT, BSD, -Apache, etc) (If you use a license badge, it should be SVG, not PNG). 
- [X] Does not include a travis badge.
- [X] Add the topics `awesome`, `awesome-list`, and `list`.

@legodude17

**Comment by legodude17** [comment]
Last thing is that some of the parts of the `Contents` section are not links. I think it would be better if they were. E.g. the parts under the `Shells` heading in the `Contents` section.

**Comment by k4m4** [comment]
That wouldn't work since those parts are not headings, and thus could not be referenced by link.
@legodude17

File: readme.md [hunk 1]
```diff
@@ -322,6 +322,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Hyper](https://github.com/bnb/awesome-hyper) - Cross-platform terminal app built on web technologies.
 - [PowerShell](https://github.com/janikvonrotz/awesome-powershell) - Cross-platform object-oriented shell.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows) - Productivity app for macOS.
+- [Terminals Are Sexy](https://github.com/k4m4/terminals-are-sexy) - Curated list for CLI lovers.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should include a short description about the list project/theme **(not the list itself)**

**Comment by sindresorhus** [comment]
> That wouldn't work since those parts are not headings, and thus could not be referenced by link.

Just remove them from the Table of Contents.

`Terminals Are Sexy` should also not be in the TOC.

---

`stackoverflow` => `Stack Overflow`

`Everything on zsh.` => `Everything on ZSH.`

`OS X` => `macOS`

`Mac OS X` => `macOS`

---


The `Other Sexy Lists` has many unrelated lists, like `awesome-python`, `awesome-sec-talks`, `public-apis`, `seclists`, `security-cheatsheets`. They are just noise. Focus on linking to relevant lists and resources.


---

Descriptions should not start with the link name:

`iTerm2 is a terminal emulator for Mac OS X that does amazing things.` => `Terminal emulator for Mac OS X that does amazing things.`

**Comment by k4m4** [comment]
All done.
- [x] Remove subheadings from the Table of Contents.
- [x] Remove `Terminals Are Sexy` from the Table of Contents.
- [x] `stackoverflow` => `Stack Overflow`
- [x] `OS X` => `macOS`
- [x] `Mac OS X` => `macOS`
- [x] Remove unrelated lists from `Other Sexy Lists`
- [x] Remove projects' names from descriptions

**Comment by sindresorhus** [comment]
Note: https://github.com/sindresorhus/awesome/pull/911/files#r112407029

**Comment by sindresorhus** [comment]
ping

**Comment by sindresorhus** [comment]
Added :)

**Comment by k4m4** [comment]
Apologies for the delayed response. Thanks for the merge! :) @sindresorhus

**Pull request merged**